Dashboard/app/cljs/cp-html.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+OUT_ROOT=../../../GAE/war/admin/frames/
+
+mkdir -p ${OUT_ROOT}
+
+if [[ $* == *--production* ]]; then
+    cat users.html | sed 's/users\.css/users\.min\.css/' > ${OUT_ROOT}users.html
```

Comment:
There is no need to `cat <file> | sed '<expr>'`, simply just used `sed` in the original file: `sed '<expr>' <file>`